src/api/auth.c

```diff
@@ -82,13 +82,15 @@ int check_client_auth(struct ftl_conn *api, const bool is_api)
 	// This may be allowed without authentication depending on the configuration
 	if(!config.webserver.api.localAPIauth.v.b && is_local_api_user(api->request->remote_addr))
 	{
+		api->message = "no auth for local user";
```

Comment:
Maybe it's just me, but reading this message I got the impression that authorization was denied for the local user.

```suggestion
		api->message = "no auth required for local user";
```